fnet/cli/train_model.py

```diff
@@ -114,6 +114,9 @@ def main(args: Optional[argparse.Namespace] = None) -> None:
     print('*** Training options ***')
     pprint.pprint(vars(args))
 
+    if not os.path.exists(args.path_save_dir):
```

Comment:
`Path.mkdir(exist_ok=True)`

Reply:
I don't believe it's necessary to use `pathlib`. It's just one option.